smoke-tests/Gemfile

```diff
@@ -8,3 +8,4 @@ gem "rspec"
 gem "pry-byebug"
 gem "aws-sdk-core", "~> 3.59"
 gem "aws-sdk-iam", '~> 1.3'
+gem "aws-sdk-route53", '~> 1.27.0'
```

Comment:
Do you specifically need `1.27.0`? If you change this to `1.27` you'll get point releases whenever you run `bundle upgrade`